src/Hl7.Fhir.Shared.Tests/TrainingTests/Z-TrainingUnitTests.cs

```diff
@@ -135,7 +135,7 @@ public void ParseElementDefinitionConstraint()
                 {
                     Expression = expression,
                     Key = key,
-                    Severity = severity == "Error" ? ConstraintSeverity.Error : ConstraintSeverity.Warning,
+                    Severity = severity == "Error" ? Hl7.Fhir.Model.ConstraintSeverity.Error : Hl7.Fhir.Model.ConstraintSeverity.Warning,
```

Comment:
ConstraintSeverity is/was/is/was part of ElementDefinition, and we cannot introduce this breaking change. That was a mistake in the pre-publication version of the generator which has since been fixed by Marco. We need to revert this change.

Reply:
Then this issue is still current in the build - I get compile errors without it.

Reply:
By pulling `develop` after the 5.1 release, this should be fixed.